.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         database: [ mysql, mongo ]
         use-dama: [ 1 ]
         use-migrate: [ 0 ]
+        use-phpunit-extension: [ 0 ]
```

Comment:
in a further PR I'll simplify this whole matrix:
 - Symfony 7.0 is not maintained anymore
 - I think we can use a default value for `use-dama`, `use-migrate`, `phpunit` and `use-phpunit-extension`, this will prevent the need to specify them in every permutation